src/Raven.Studio/typescript/components/pages/database/settings/tombstones/TombstonesState.tsx

```diff
@@ -0,0 +1,261 @@
+import React, { useEffect, useState } from "react";
+import { ShardedViewProps } from "components/models/common";
+import { useServices } from "components/hooks/useServices";
+import { useAsync, useAsyncCallback } from "react-async-hook";
+import { LoadError } from "components/common/LoadError";
+import VirtualGrid from "components/common/VirtualGrid";
+import textColumn from "widgets/virtualGrid/columns/textColumn";
+import virtualColumn from "widgets/virtualGrid/columns/virtualColumn";
+import virtualGridController from "widgets/virtualGrid/virtualGridController";
+import SubscriptionInfo = Raven.Server.Documents.TombstoneCleaner.TombstonesState.SubscriptionInfo;
+import { Alert, Card } from "reactstrap";
+import ButtonWithSpinner from "components/common/ButtonWithSpinner";
+import useConfirm from "components/hooks/useConfirm";
+import { FlexGrow } from "components/common/FlexGrow";
+import { AboutViewHeading } from "components/common/AboutView";
+import { Icon } from "components/common/Icon";
+
+export default function TombstonesState({ db, location }: ShardedViewProps) {
+    const { databasesService } = useServices();
+
+    const asyncGetTombstonesState = useAsync(() => databasesService.getTombstonesState(db, location), []);
+    const asyncForceTombstonesCleanup = useAsyncCallback(() => databasesService.forceTombstonesCleanup(db, location));
+
+    const [collectionsGrid, setCollectionsGrid] = useState<virtualGridController<TombstoneItem>>();
+    const [subscriptionsGrid, setSubscriptionsGrid] = useState<virtualGridController<SubscriptionInfo>>();
+
+    const tombstonesAlert = () => (
+        <Alert color="info" className="mb-0">
+            <p>
+                By default, tombstones cleanup is scheduled to be carried out by the server every 5 minutes, unless
+                configured otherwise.
+            </p>
+            <p className="mb-0">
+                Upon clicking <strong>Force Cleanup</strong>, the action will be executed immediately - any tombstone
+                that can be removed will be deleted.
+            </p>
+        </Alert>
+    );
```

Comment:
It should be a separate component

Reply:
The alert has been moved to a separate component